Symfony/CS/Fixer/Symfony/NamespaceNoLeadingWhitespaceFixer.php

```diff
@@ -72,6 +72,10 @@ public function getDescription()
 
     private static function endsWithWhitespace($str)
     {
-        return '' !== $str && ctype_space(substr($str, -1));
+        if ('' === $str) {
+            return false;
+        }
+
+        return '' === trim(substr($str, -1));
```

Comment:
I thouth about that - but here you will trim all whitespaces on the right side.
On my PR you will check only one char.